packages/simulation-new/src/common/actors.tsx

```diff
@@ -131,50 +145,58 @@ export namespace Sensor {
   export const make = ({
     pos,
     id,
-  }: Pos.Type & {
-    id?: string;
-  }): Type => ({
-    t: "Sensor",
-    pos,
-    ...Id.make(id),
-    water: 100,
+    water,
+  }: Partial<ActorBase & { water: number }>): Type => ({
+    t: "Plant",
+    pos: pos || Pos.makeRandom().pos,
+    id: id || Id.make().id,
+    water: water || 100,
   });
 
-  export namespace Actions {
-    export type SetWaterLevel = z.TypeOf<typeof SetWaterLevel>;
-    export const SetWaterLevel = z.object({
-      t: z.literal("SetWaterLevel"),
-      value: z.number(),
-    });
-
-    export type WaterPlant = z.TypeOf<typeof WaterPlant>;
-    export const WaterPlant = z.object({
-      t: z.literal("WaterPlant"),
-      sensorId: z.string(),
-    });
-
-    export const Cancel = z.null();
-
-    export type Actions = z.TypeOf<typeof Actions>;
-    export const Actions = SetWaterLevel;
-
-    export const apply = (
-      _: Actor.ReadonlyActorsMap,
-      robot: Type,
-      action: Actions
-    ) => {
-      robot.water = action.value;
-    };
+  // export namespace Actions {
+  //   export type SetWaterLevel = z.TypeOf<typeof SetWaterLevel>;
+  //   export const SetWaterLevel = z.object({
+  //     t: z.literal("SetWaterLevel"),
+  //     value: z.number(),
+  //   });
+
+  //   export type WaterPlant = z.TypeOf<typeof WaterPlant>;
+  //   export const WaterPlant = z.object({
+  //     t: z.literal("WaterPlant"),
+  //     plantId: z.string(),
+  //   });
+
+  //   export const Cancel = z.null();
+
+  //   export type Actions = z.TypeOf<typeof Actions>;
+  //   export const Actions = SetWaterLevel;
+
+  //   export const apply = (
+  //     _: ActorData.ReadonlyActorsMap,
+  //     robot: Type,
+  //     action: Actions
+  //   ) => {
+  //     robot.water = action.value;
+  //   };
+  // }
```

Comment:
I'd just remove this and bring it from Git if we need it

Reply:
forgot to remove it